rsr_convert_scripts/viirs_rsr.py

```diff
@@ -263,4 +254,6 @@ def main():
 
 if __name__ == "__main__":
     LOG = logging.getLogger('viirs_rsr')
```

Comment:
I just noticed, this logger is already created at the top of the script isn't it? Can this be removed?

Reply:
Yes, right, thanks!